client/stylesheets/_toCat.scss

```diff
@@ -2436,21 +2454,56 @@ i:hover {
     }
 }
 
+.headerDropDown ul li:first-child {
+	border-top: none !important;
+	margin-top: 10px;
+}
 
+.headerDropDown ul li:last-child {
+	border-bottom: none !important;
+	margin-bottom: 10px;
+} 
+
+.clickDropper li ul {
+	margin-left: -180px;
+}
+
+.clickDropper li ul li:first-child::before {
+	color: #404040;
+   content: '▲';
+   float: right;
+   margin: -50px -43px 0 0;
+   font-size: 1.3rem;
+}
+
+.clickDropper li ul li a {
+	padding: 15px 20px 15px 15px !important;
+}
 
 // .headerDropDown ul 
 
+
+.hoverDropper {
+	padding: 12px !important;
+}
+
+.hoverDropper:hover {
+	color: #fff !important;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs<br>Properties should be ordered background-color, color